angr/analyses/slice_to_sink/slice_to_sink.py

```diff
@@ -0,0 +1,132 @@
+from functools import reduce
+
+from .transitions import merge_transitions
+
+
+class SliceToSink():
+    """
+    The representation of a slice of a CFG.
+    """
```

Comment:
Also I kinda dislike the name of this class. I like `Slice` or `CFGSlice` better. `SliceToSink` doesn't sound like a data class to me.

Reply:
This is not meant to represent "any" slice of a CFG, but a slice leading to a target, and that's the context I am using it in.
(I did not try to abstract and design it for other uses)
For that matter, I find `SliceToSink` more explicit than `Slice` or `CFGSlice`...

Reply:
Sure...

`SliceToSink` just sounds like a verb: _slice_ to a sink.

Reply:
`CFGSliceToSink` seems even more accurate, and probably sounds less than an action.

Reply:
Sounds good to me.